atmospheric_explorer/plotting_apis.py

```diff
@@ -188,19 +193,13 @@ def line_with_ci_subplots(
         hovermode="closest",
     )
     fig.update_traces(mode="lines+markers")
-    fig.update_traces(
-        selector=-2 * len(countries) - 1, showlegend=True
-    )  # legend for ci
+    fig.update_traces(selector=-2 * len(labels) - 1, showlegend=True)  # legend for ci
     fig.update_traces(selector=-1, showlegend=True)  # legend for ci
     return fig
 
 
 def _ghg_surface_satellite_yearly_data(
```

Comment:
I'm not sure what you mean, this fetches the data needed for the ghg

Reply:
I also don't know what I meant 🤪 I was probably seeing some changes not very clearly from the GitHub mobile interface.

Anyway, slightly connected to this comment: we only have a separate function for getting data vs. creating the plot for GHG. Is it definitely necessary because this dataset is more complex than EAC4? 
Better yet: should we have two separate functions for getting data and creating the plot for all EAC4 plots as well? We could do this as a simple step towards API modularization, even without creating classes. Of course, it would be a refactoring task that's unrelated to this PR. Do you think we should do this?